tianshou/policy/modelfree/ppo.py

```diff
@@ -34,6 +34,8 @@ class PPOPolicy(PGPolicy):
         defaults to ``True``.
     :param bool reward_normalization: normalize the returns to Normal(0, 1),
         defaults to ``True``.
+    :param int max_batchsize: the maximum number of batchsize when computing
```

Comment:
I guess `maximum number of batchsize` is not what you mean. It should be `the maximum size of the batches` no ?

Reply:
exactly

Reply:
`max_batchsize` is fine to me ! Just the description `the maximum number of batchsize` that is not :smile: